data/pages-content/accessibility.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Accessibility at EuroPython
+subtitle: Information about Accessibility at EuroPython as an attendee with maps of venues, childcare and other accessibility information.
+---
+
+# Accessibility at EuroPython
+
+EuroPython ensures that it’s welcoming, accessible and inclusive for all our guests attending the conference. If you have suggestions to improve accessibility, let us know by contacting us at [helpdesk@europython.eu](mailto:helpdesk@europython.eu).
+
+## General
+For enquiries, visit our helpdesk at registration  or look for volunteers wearing Yellow conference t-shirt when enquiring about accessibility which includes:-
+
+* Lifts 
+* Accessible toilet facilities
+* Dietary requirements
+
+At main talk rooms, enquire with the session chair before the talks about designated seating, maintaining clear pathways, or other needs.
+
+## Feeding your child
+The conference venue (The Prague Congress Centre) and sprints venue (VŠE) will both have quiet rooms (see below for locations).  If you require to breastfeed your child, if preferred. Please note, there are no restrictions on where you can breastfeed your child.
+
+## Dietary Requirements
```

Comment:
We have some information from the venue. I don't know what they want to do with the Clubs area during the tutorial days, but they will set up dietary station the bar on the second floor foyer. People who need special diet, e.g. vegan, sugar, garlic free can pick up their food there. And allergen information will be displayed on the menu for people who prefer any food restrictions.